callback.php

```diff
@@ -41,6 +41,10 @@
           
             $jwt = new XeroAPI\XeroPHP\JWTClaims();
             $jwt->setTokenId($accessToken->getValues()["id_token"]);
+
```

Comment:
Is this for troubleshooting or does it need to be added to the sample app for all?